a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -277,8 +268,14 @@ def get_new_lb_networking_subflow(self, topology):
             requires=(a10constants.VTHUNDER, constants.AMPHORA)))
         new_LB_net_subflow.add(vthunder_tasks.EnableInterface(
             requires=a10constants.VTHUNDER))
-
+        new_LB_net_subflow.add(a10_database_tasks.MarkVThunderStatusInDB(
+            name="set_vThunder_entry_active",
+            requires=a10constants.VTHUNDER,
+            inject={"status": constants.ACTIVE}))
         if topology == constants.TOPOLOGY_ACTIVE_STANDBY:
+            new_LB_net_subflow.add(vthunder_tasks.CreateHealthMonitorOnVThunder(
+                name='creating health monitor to master vthunder',
+                requires=(a10constants.VTHUNDER)))
```

Comment:
This requires shouldn't be wrapped in a tuple: `requires=(a10constants.VTHUNDER)` should be `requires = a10constants.VTHUNDER`